src/group_theory/abelianization.lean

```diff
@@ -106,4 +109,13 @@ theorem hom_ext (φ ψ : abelianization G →* A)
   (h : φ.comp of = ψ.comp of) : φ = ψ :=
 monoid_hom.ext $ λ x, quotient_group.induction_on x $ monoid_hom.congr_fun h
 
+/-- Equivalent groups have equivalent abelianizations -/
+def mul_equiv.abelianization_congr {G H : Type*} [group G] [group H] (e : G ≃* H) :
```

Comment:
Hmm, this way it becomes `abelianization.mul_equiv.abelianization_congr`, but this is not your intention in <https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there.20code.20for.20X.3F/topic/injections.20to.20equiv/near/268040127>, @eric-wieser , is it? Should I move it after the `end abelianization`?

Reply:
I’ll guess “yes” and move it out of the namespace.

Reply:
You can also put `_root_` at the start of the name to escape the current namespace(s)